am-lib/src/main/java/uk/gov/hmcts/reform/amlib/exceptions/ErrorAddingEntriesException.java

```diff
@@ -1,9 +1,9 @@
 package uk.gov.hmcts.reform.amlib.exceptions;
 
-public class ErrorAddingEntriesToDatabaseException extends RuntimeException {
+public class ErrorAddingEntriesException extends RuntimeException {
     private static final long serialVersionUID = 1L;
 
-    public ErrorAddingEntriesToDatabaseException(Throwable throwable) {
+    public ErrorAddingEntriesException(Throwable throwable) {
```

Comment:
What about having one exaction class that covers all exceptions around data persistence e.g. `PersistenceException`.